_posts/2017/2017-07-21-fullstack-london-of-dragons-and-numbers.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Fullstack London - Of dragons, numbers and IoT"
+date: "2017-07-21 06:34:02"
+image: blog-header/fullstack.jpg
+categories: events
+authors: ["Erik"]
+---
+
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+After one year of abscence I was attending [Fullstack](https://skillsmatter.com/conferences/8264-fullstack-2017-the-conference-on-javascript-node-and-internet-of-things) London again. The past years were well-organised, fun and had an awesome speaker deck on top of that.
+This year was no difference.
+
+Here are some things I took away from these awesome three days.
+
+## Keynotes and Workshops
+
+The opening keynote was held by Douglas Crockford (the father of JSON) himself.
+While talking about numbers and their history he came up with [DEC64](http://dec64.com/) a new number type that fixes truncation and overflow problems (besides others) originated from the fifties.
+The session was not only informative but also very humorous as Crockford is often very opinionated and does not try to hide that at all.
+On top of that Skills Matter gifted (among others) Crockfords _The Good Parts_ book which he signed during the conference.
+
+Later that day [Amie D. Dansby](https://twitter.com/amiedoubleD) (who somehow reminded my of NCIS' Abby) an Iot enthusiast and maker (with an RFID chip in her hand) gave an inspiring talk about releasing your inner maker.
+IMHO her slides where a bit overloaded but her spirit and message as well as the content totally made up for that!
+She was also the one giving a hands on IoT workshop with Onion's [Omega2](https://onion.io) development board.
+After DDoS'ing the conferences Wi-Fi frequencies and some struggle for more space on the board I managed to get a working Twitter wearable displaying my latest tweet on a OLED display.
+
+There were several other inspiring keynotes including Sitepen's Dylan Schiemann (wearing a Dojo 2.0 t-shirt) about the state of JavaScript and Myles Borins talking about how Node.js releases work.
```

Comment:
SitePen's Dylan Schiemann (https://twitter.com/dylans?lang=en)

Myles Borins > is that the right account? https://twitter.com/mylesborins?lang=en